recipe/meta.yaml

```diff
@@ -31,14 +31,11 @@ requirements:
 
 test:
   files:
-    - ice_skating.py
-    - male1.txt
-    - import_test.py
-    - test.sh
+    - tests/
```

Comment:
Moving these breaks the test because the ice_skating file looks for the male1.txt file in the directory that the script was executed in.